CMakeLists.txt

```diff
@@ -5,14 +5,14 @@
 cmake_minimum_required(VERSION 3.5)
 set(CMAKE_CXX_STANDARD 14)
 
-project(walking-controllers)
+## MAIN project
+project(WalkingControllers
+  VERSION 0.0.1)
```

Comment:
Please modify the version once https://github.com/robotology/walking-controllers/pull/44 is merged.

Reply:
Yes sure. Since the structure of the project is completely changed the version should be `0.3.0` right?

Reply:
I agree.

Reply:
Done!